pennylane/templates/subroutines/trotter.py

```diff
@@ -132,6 +133,17 @@ def my_circ():
     .. details::
         :title: Usage Details
 
+        An *upper-bound* for the error in approximating time-evolution using this operator can be
+        computed by calling the error function. It is computed using two different methods; the
```

Comment:
Please change to :func:`~.TrotterProduct.error()` or something similar.

Reply:
Added